server/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const bodyparser = require('body-parser');
+const db = require('../database/index.js');
+const Promise = require('bluebird');
+const request = require('request');
+const Comments = require('../database/comments.js');
+
+let app = express();
+app.use(bodyparser.json());
+app.use(express.static(__dirname + '/../client/dist'));
+
+
+
+app.post('/recipe', function (req, res) {
+
+  res.send('hi');
+})
+
+
+app.get('/recipe', function (req, res) {
+  return Comments.fetch()
```

Comment:
you seem to be getting and returning to the client the entire contents of your db on each get request. not a big deal when we are looking at a small db, but not good practice. The data you are getting from the server should be specific to the page you are on.